code/go/0chain.net/miner/worker.go

```diff
@@ -218,7 +218,8 @@ func (mc *Chain) getPruneCountRoundStorage() func(storage round.RoundStorage) in
 }
 
 func (mc *Chain) MinerHealthCheck(ctx context.Context) {
-	const HEALTH_CHECK_TIMER = 60 * 5 // 5 Minute
+	// TODO: Move that to a unified config file.
+	const HEALTH_CHECK_TIMER = 1 * time.Minute // 5 Minute
```

Comment:
it's one minute

Reply:
what is changed here?

Reply:
I set it down to 1 minute for testing. Fixed.